src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/CaseNoteService.kt

```diff
@@ -29,4 +32,9 @@ class CaseNoteService(
     )
     return caseNoteRepository.save(caseNote)
   }
+
+  fun findByReferral(referralId: UUID, pageNumber: Int, pageSize: Int): Page<CaseNote> {
+    var pageRequest = PageRequest.of(pageNumber, pageSize, Sort.by("sentAt").ascending())
+    return caseNoteRepository.findAllByReferralId(referralId, pageRequest)
```

Comment:
Should this not be ordered by some field as without SQL doesn't guarantee in which order they are returned. The definition of a page row should be determined by some sort of line number shouldn't it.

Reply:
sorry, I don't understand the question? It is ordered by SentAt, this value is never null, it never changes, will be unique and new case notes would be appended to the end of this ordering.

Reply:
re-read your question!

So the sentAt time will always be different since a user can only create one case note at a time.